src/MaIN.Services/Services/LLMService/OpenAiCompatibleService.cs

```diff
@@ -267,6 +269,18 @@ await _notificationService.DispatchNotification(
         }
     }
 
+    protected virtual async Task<LLMTokenValue?> ProcessChatCompletionChunk(string data)
```

Comment:
accidently came across this, Is there any reason that this is async? Saw this also in deepseek implementation, and there is also no asynchronous calls there. Is there an implementation of this that requires async?